src/support/azext_support/_validators.py

```diff
@@ -134,10 +135,31 @@ def _check_name_availability_no_subscription(cli_ctx, resource_name, resource_ty
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
 
-
 def _check_name_availability_no_subscription_ticket(cli_ctx, ticket_name, resource_name, resource_type):
     from .aaz.latest.support.no_subscription.tickets import CheckNameAvailability
     check_name_availability_input = {"support_ticket_name": ticket_name, "name": resource_name, "type": resource_type}
     resp = CheckNameAvailability(cli_ctx=cli_ctx)(command_args=check_name_availability_input)
     if not resp["nameAvailable"]:
         raise CLIError(resp["message"])
+
+def validate_file_path(file_path):
+    if not os.path.exists(file_path):
+        raise CLIError("File does not exist!")
```

Comment:
Output the file path also in the error message

Reply:
I spoke with Anagh about this in the past, and this is the error message we have agreed upon. Will not fix